docs/build-on-linea/hackathons.md

```diff
@@ -15,123 +15,97 @@ All winners have the opportunity to bring their project to fruition through a va
 
-_Bonus points_
-- Incorporating [partner tooling](https://docs.linea.build/build-on-linea/tooling) to make more robust and interesting examples
-- A frog is somewhere in your submission
+**What are examples of acceptable work**
+- A step-by-step, published tutorial on how to build a (maybe frog-themed :frog:) dapp on Linea
+- A translated guide on how to build on Linea
+- A published video explainer on how to build with Linea or Linea concepts
+- Improving the UI of a hackathon project you built before
+- Flappy bird, but with frogs :frog: and on-chain
 
-_Judging criteria_
-- UX/UI
-- Utility
-- Technicality
-- Market readiness
+**What is the judging criteria**
+- UX - how easy is it for a user to navigate or how easy it is to follow the tutorial
+- Utility - how useful is your project
+- Technicality - how complex was this project to build
+- WOW factor - a catch-all vibe check for projects that are simply fun to interact with or pushing the boundaries of on-chain use cases
 
-## IRL hackathons
+**What counts as a qualified submission**
 
-### EthVietnam
+If you're submitting a dapp or Farcaster frame...
+- Open source, public Github repo
+- Non-trivial progress made on your project during that month (as measured by your Github commit history)
+- Verified smart contract deployed on Linea Sepolia
+- Working product demo link
+- Short video explainer (less than 5 minutes) of what your project is and what changes you made that month if it is not a new project
+- Posted and tagged on Twitter ([@lineabuild](http://x.com/lineabuild)) or Farcaster ([@linea](http://warpcast/linea))
 
-**Build a defi, deso, gaming, or frog-themed dapp on Linea ($5k pool prize)**
+If you're submitting a tutorial...
+- Open source, public Github repo
+- Published tutorial link
+- Non-trivial progress made on your project during that month (as measured by your Github commit history)
+- Bonus points for publishing an accompanying video explainer
+- Posted and tagged on Twitter ([@lineabuild](http://x.com/lineabuild)) or Farcaster ([@linea](http://warpcast/linea))
 
-We’re looking for the novel ideas expanding the adoption of web3 and want to connect with hackers, giving them exclusive invites to our Linea Builders Club, where you can find tech talks by our ZK cryptographers, private mini hack opportunities, educational programs, and more!
+Ultimately, a "high-quality" submission is quite subjective, but you can imagine invalid submissions as something like unusable demos, projects that are simply slightly modified clones of other projects, and otherwise scrapped together hacks that took little to no effort to put together.
 
-We're judging based on technicality, utility, UX, and market readiness - bonus points if you add a frog theme or use any [partner tooling](https://docs.linea.build/build-on-linea/tooling)
+**How do I submit**
 
-*Requirements*
+Find the submission deform linked under the current month.
```

Comment:
Maybe a stupid question but what's a deform? If it's a platform/service maybe it needs to be capitalised